src/drivers/video/vbe/vbe_video.c

```diff
@@ -0,0 +1,21 @@
+#include "vbe_video.h"
```

Comment:
Could we name this file `src/drivers/vbe_video.c`? (and same for the header file)

Reply:
Done!